src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: getMaxWidth(
+    $day-box-size, $day-box-gap, $day-box-paddings
+  );
+  max-width: getMaxWidth(
+    $day-box-size, $day-box-gap, $day-box-paddings
+  );
```

Comment:
I am not sure why you assign both `min-width` and `max-width` to the same value. I think setting only the `width` would suffice